src/dbnode/client/options.go

```diff
@@ -32,16 +32,15 @@ import (
 	"github.com/m3db/m3/src/dbnode/encoding/m3tsz"
 	"github.com/m3db/m3/src/dbnode/encoding/proto"
 	m3dbruntime "github.com/m3db/m3/src/dbnode/runtime"
+	"github.com/m3db/m3/src/dbnode/storage/namespace"
```

Comment:
Hm it feels like time to move this package out from `storage`. What do you reckon?

/cc @robskillington @richardartoul

Reply:
Fair, yeah I agree.

Maybe we should follow up with a single atomic commit we can do this with (otherwise it might make for painful rebases while this PR is alive)?

Reply:
I agree. I will do another PR just to move namespace out of storage aftr this PR.

Reply:
@Haijuncao can you open an issue to track this?  We should make sure we remember to do this relatively immediately.

Reply:
https://github.com/m3db/m3/issues/1599

I proposed an alternative in the issue. please comment. thanks.